apron/rationale.texi

```diff
@@ -84,7 +84,7 @@ numerical variables are programmed in C or C++.
 
 An @sc{OCaml} version is already available. The interface between
 OCaml and C is even generic and any libraries can benefit from it by
-providing the glue for just one function (see XX).
```

Comment:
If someone tells me where the broken reference is meant to point, I can fix it instead of deleting it.

Reply:
Actually, a C library need only to export a function creating an `ap_manager_ptr`. All the functions registered in the manager are then called through `abstract0_` or `abstract1_` functions, which are in Apron.
This is similar for OCaml. The OCaml binding for a new library needs only export in OCaml the manager creation function.
I think that this is not explained in the documentation at the moment.
I suggest we replace "see XX" with "the function creating a manager".

Reply:
The suggestion seems good to me.
Done in  a5c6ba6 .